src/components/Charts/utils.ts

```diff
@@ -238,32 +238,49 @@ export const getChartRegions = (
   },
   {
     valueFrom: zones[Level.MEDIUM].upperLimit,
+    valueTo: zones[Level.MEDIUM_HIGH].upperLimit,
+    name: zones[Level.MEDIUM_HIGH].name,
+    color: zones[Level.MEDIUM_HIGH].color,
+  },
+  {
+    valueFrom: zones[Level.MEDIUM_HIGH].upperLimit,
     valueTo: maxY,
     name: zones[Level.HIGH].name,
     color: zones[Level.HIGH].color,
   },
 ];
 
+const isBetween = (zoneLow: LevelInfo, zoneHigh: LevelInfo, value: number) =>
+  zoneLow.upperLimit <= value && value < zoneHigh.upperLimit;
+
 export const getZoneByValue = (value: number, zones: Zones) => {
   if (value < zones[Level.LOW].upperLimit) {
     return zones[Level.LOW];
   }
-  return value > zones[Level.MEDIUM].upperLimit
-    ? zones[Level.HIGH]
-    : zones[Level.MEDIUM];
+
+  if (isBetween(zones[Level.LOW], zones[Level.MEDIUM], value)) {
+    return zones[Level.MEDIUM];
+  }
+
+  if (isBetween(zones[Level.MEDIUM], zones[Level.MEDIUM_HIGH], value)) {
+    return zones[Level.MEDIUM_HIGH];
+  }
+
+  return zones[Level.HIGH];
```

Comment:
I _think_ that this code almost duplicates https://github.com/covid-projections/covid-projections/pull/850/files?file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#diff-f81cf374a211ca4db3c5e39b93306773R65, I will look into that after we merge to see if we can centralize the logic (@mikelehen 's code is much cleaner)